GVFS/GVFS.Mount/InProcessMountVerb.cs

```diff
@@ -47,6 +47,14 @@ public InProcessMountVerb()
             HelpText = "Show the debug window.  By default, all output is written to a log file and no debug window is shown.")]
         public bool ShowDebugWindow { get; set; }
 
+        [Option(
+            's',
+            GVFSConstants.VerbParameters.Mount.StartedByService,
+            Default = "false",
+            Required = false,
+            HelpText = "Service initiated mount.")]
+        public string StartedByService { get; set; }
+

```

Comment:
Arguably this isn't consistent with MountVerb which allows you to pass a series of internal parameters.  On the InProcessMountVerb itself, you only need 1 parameter so I'd rather not make it take a series until/unless the need arises.

Reply:
The `MountVerb` gets called in-proc by other verbs. No such requirement here.